prow/cip/config.yaml

```diff
@@ -0,0 +1,153 @@
+# Basically anything with a "name: ..." and "spec: ..." is a Prow Job. Prow
+# Jobs can be triggered periodically if they are under "periodics" or
+# automatically based on PRs being opened (presubmits) or after they are merged
+# "postsubmits".
+#
+# To update this, first run:
+#
+# bazel run //prow/cmd/checkconfig -- --plugin-config=PATH/TO/cip-infra/plugins.yaml --config-path=PATH/TO/cip-infra/config.yaml
+#
+# Then run (for plugins)
+#
+# kubectl create configmap plugins \
+#  --from-file=plugins.yaml=plugins.yaml --dry-run -o yaml | kubectl replace configmap plugins -f -
+#
+# Then run (for config)
+#
+# kubectl create configmap config \
+#  --from-file=config.yaml=config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+
+plank:
+  default_decoration_config:
+    timeout: 7200000000000 # 2h
+    grace_period: 15000000000 # 15s
+    utility_images:
+      clonerefs: "gcr.io/k8s-prow/clonerefs:v20190125-2aca69d"
+      initupload: "gcr.io/k8s-prow/initupload:v20190125-2aca69d"
+      entrypoint: "gcr.io/k8s-prow/entrypoint:v20190125-2aca69d"
+      sidecar: "gcr.io/k8s-prow/sidecar:v20190125-2aca69d"
+    gcs_configuration:
+      # GCS Bucket created specifically for the demo
+      bucket: "cip-demo-staging"
+      path_strategy: "explicit"
+      default_org: "cip-bot"
+      default_repo: "cip-manifest"
+    # To create the secret, download a .json service account key from Pantheon,
+    # then run
+    # kubectl create secret generic cip-service-account --from-file=service-account.json=<PATH_TO_KEY_FILE.JSON>
+    # The entry here is the Secret name
+    # (kubectl will say `secret "cip-service-account" created`)
+    #
+    gcs_credentials_secret: "cip-service-account"
+
+periodics:
+# Daily run, to enforce the state of the manifest on a regular basis. This could
+# happen more frequently (perhaps hourly), but it's probably not a big deal,
+# given how the postsubmit will take care of all needs (in theory). This is more
+# of a sanity check in cases where there are no merges to "master" branch for
+# long periods of time.
+- interval: 24h
+  max_concurrency: 1
+  agent: kubernetes
+  # This name is the "job name", passed in as "--job=NAME" for mkpj.
+  name: cip-sanity
+  labels:
+    mount-service-account-key: "true"
+  # Enable Pod Utilities. See https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md.
+  decorate: true
+  extra_refs:
+  # Because of Pod Utilities, we clone the below repo automatically, and, get
+  # dropped into /home/prow/go/src/github.com/cip-bot/cip-manifest.
+  - org: cip-bot
+    repo: cip-manifest
+    base_ref: master
+  spec:
+    containers:
+    - &cip_prod
```

Comment:
hehe @fejta will fight you for this :^)

Reply:
Ack. Will remove YAML anchors/references.